bugbear.py

```diff
@@ -495,6 +496,7 @@ def visit_For(self, node):
         self.check_for_b020(node)
         self.check_for_b023(node)
         self.check_for_b031(node)
+        self.check_for_b999(node)
```

Comment:
```suggestion
        self.check_for_b038(node)
```
We generally just go next number unless we make optional - They are >900.